python/docs/source/guide/installation.md

```diff
@@ -5,7 +5,7 @@ We suggest using 3.11 or newer, since that provides more precise error locations
 
 ```{code-block} bash
 :caption: Installing Kaskada
-pip install kaskada>=0.6.0-a.0
+pip install kaskada>=0.6.0-a.3
```

Comment:
FWIW, the `>=0.6.0-a.0` should still pick the "newest" version greater than that, so we shouldn't need to mention `a.3` specifically. The fact it includes `-a.0` means it will allow pre-release versions.

This should all be simplified once we call this `0.6.0` and start using non-pre-release versions. Then we could just say `pip install>=0.6.0` and rely on the version resolution to ensure it gets the newest (the greater than equal just ensures it doesn't install one of the "old" clients).